tests/cypress/integration/masthead_test.spec.js

```diff
@@ -0,0 +1,70 @@
+import ListingPage from "../support/pages/admin_console/ListingPage.js";
+import LoginPage from "../support/pages/LoginPage.js";
+import SidebarPage from "../support/pages/admin_console/SidebarPage.js";
+
+const loginPage = new LoginPage();
+
+const logOutTest = (userDropdown) => {
+  it("logs out", () => {
+    cy.get(userDropdown).click();
+    cy.get("#sign-out").click();
```

Comment:
```suggestion
    headerpage.signOut();
```

And modify the signOut() method to user a different dropdown according to the viewport wide 
e.g. `Cypress.config().viewportWidth < 768`